ansible/idrsystems-playbooks/idr-analysis.yml

```diff
@@ -34,3 +34,9 @@
   - role: openmicroscopy.versioncontrol-utils
 
   - role: openmicroscopy.celery-docker
+
+- hosts: idr-analysis
+  tasks:
+  - file:
+      path: /scratch
+      mode: "1777"
```

Comment:
Not sure, since it's already there though not being used. I guess either remove it or leave the playbooks unchanged?